pages/success-banner.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Head from 'next/head';
+
+class Button extends React.Component {
+  render() {
+    return (
+      <a
+        href={ this.props.href }
+        target="_blank"
+        className="button">
+        { this.props.children }
+      </a>
+    )
+  }
+}
+
+const Banner = () => (
+  <main className="vertical center">
+    <div>
+      <h1>We hope you enjoyed using</h1>
+      <img src="/static/etcher.svg" className="etcher-logo"></img>
```

Comment:
Should the `<img>` tag be _inside_ the `<h1>` tag? https://stackoverflow.com/questions/1605454/can-we-place-img-inside-h1-according-to-web-standards